arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/DefaultDeploymentScenarioGenerator.java

```diff
@@ -92,20 +91,10 @@
 
         scanner.scanPackage();
 
-        Set<String> prefixes = codeSources.stream().map(e -> e.getLocation().toExternalForm()).collect(Collectors.toSet());
```

Comment:
If we remove `prefixes`, we can also remove `codeSources` and everything related, as that's only used for this filtering.

Reply:
@Ladicek I believe I will be able to remove the `try { codesources.add() }` block but keep the rest (as that is used for preparing the archive).